model_logging/migrations/0001_initial.py

```diff
@@ -1,8 +1,13 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
-
 from django.db import models, migrations
-import pgcrypto.fields
+from unittest.mock import Mock
+
+try:
+    import pgcrypto.fields
+except ImportError:
+    pgcrypto = Mock()
```

Comment:
That's pretty slick. No objection from me, if it works that's sweet, and nicely solved!

@Minglee01 If you wouldn't mind copying basically Todd's exact post into the migration as a comment, that'd be awesome.